src/controllers/commands/CommandController.cpp

```diff
@@ -938,6 +938,110 @@ void CommandController::initialize(Settings &, Paths &paths)
         return "";
     });
 
+    auto formatModsError = [](HelixGetModeratorsError error, QString message) {
+        using Error = HelixGetModeratorsError;
+
+        QString errorMessage = QString("Failed to get moderators: ");
+
+        switch (error)
+        {
+            case Error::Forwarded: {
+                errorMessage += message;
+            }
+            break;
+
+            case Error::UserMissingScope: {
+                errorMessage += "Missing required scope. "
+                                "Re-login with your "
+                                "account and try again.";
+            }
+            break;
+
+            case Error::UserNotAuthorized: {
+                errorMessage +=
+                    "Due to Twitch restrictions, this command can only be "
+                    "used by moderators. To see the list of mods you must "
+                    "use the Twitch website.";
+            }
```

Comment:
```suggestion
            case Error::UserNotAuthorized: {
                errorMessage +=
                    "Due to Twitch restrictions,"  //
                    "this command can only be used by channel moderators. "
                    "To see the list of mods you must use the Twitch website.";
            }
```
Inserted the word `channel` and tricked the formatter so the message is easy to read via the code

Reply:
For me it looks like in the suggested change there will be a space missing after the comma (`restrictions,this command can only  be used`). I think it should be (Taking the suggestion as base)
````
                errorMessage +=
                    "Due to Twitch restrictions, "  //
                    "this command can only be used by channel moderators. "
                    "To see the list of mods you must use the Twitch website.";
```

Reply:
> For me it looks like in the suggested change there will be a space missing after the comma 

Yes you were correct, resolved